lib/src/mount.rs

```diff
@@ -129,6 +129,21 @@ pub(crate) fn mount(dev: &str, target: &Utf8Path) -> Result<()> {
     )
 }
 
+pub fn unmount(target: &Utf8Path, lazy: bool) -> Result<()> {
```

Comment:
This is OK as is because we are already forking in other places but my thought lately is we should really just switch to direct syscalls.